packaging/ubuntu-trusty/debian/changelog

```diff
@@ -1,5 +1,6 @@
-amazon-ecs-init (1.14.0-2) trusty; urgency=low
+amazon-ecs-init (1.14.3-1) trusty; urgency=low
```

Comment:
If we're going to ship this file, we should probably update it as we release. Otherwise people who build packages from it will get confusing results.